modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java

```diff
@@ -92,8 +92,9 @@ public String getPortletId() {
 			throw new UnsupportedOperationException();
 		}
 
-		return "com_liferay_object_web_internal_object_definitions_portlet_" +
-			"ObjectDefinitionsPortlet_" + getObjectDefinitionId();
+		return StringBundler.concat(
+			"com_liferay_object_web_internal_object_definitions_portlet_",
+			"ObjectDefinitionsPortlet_", getCompanyId(), "_", getName());
```

Comment:
Why do we need to change the PortletId if the objectDefinitionId is unique? If is really necessary, do we need to do a upgrade process, right?